src/components/PostCard/PostHeader/PostHeader.js

```diff
@@ -89,7 +89,8 @@ export default class PostHeader extends PureComponent {
       { icon: 'Trash', label: 'Remove From Group', onClick: removePost, red: true }
     ], item => isFunction(item.onClick))
 
-    const canHaveTimes = type === 'offer' || type === 'request' || type === 'resource'
+    const typesWithTimes = ['offer', 'request', 'resource', 'project']
```

Comment:
event has times too, shouldnt that be here too? i guess it must be working differently

Reply:
the timeframe for requests/offers/projects/resources displays differently than the time of an event